app/frontend/src/components/UserChatMessage/UserChatMessage.tsx

```
@@ -5,37 +5,19 @@ import rehypeRaw from "rehype-raw";
-    setQuestion: (question: string) => void;
-    answers: any[];
-    setAnswers: (answers: any[]) => void;
-    storage: indexedDBStorage;
-    lastQuestionRef: MutableRefObject<string>;
-    current_id: number;
-    is_bot: boolean;
+    onRollbackMessage: () => void;
 }
 
-export const UserChatMessage = ({ message, token, setQuestion, answers, setAnswers, storage, lastQuestionRef, current_id, is_bot }: Props) => {
+export const UserChatMessage = ({ message, onRollbackMessage: onRollbackMessage }: Props) => {
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)